recipe/meta.yaml

```diff
@@ -44,29 +40,30 @@ outputs:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - armpl  # [aarch64]
+        - armpl  # [aarch64]        
+        - ninja-base
+        - pkg-config
       host:
         - python
         - pip
-        - cython >=0.29.34,<3.0
-        - setuptools <60.0.0    # [py<=311]
-        - wheel                 # [py<=311]
-        - meson-python >=0.10.0 # [py>=312]
-        - meson >=1.1.0         # [py>=312]
+        - cython >=0.29.34,<3.1
+        - python-build
+        # dependencies of vendored meson-python
```

Comment:
When I look at the dependencies of meson-python on defaults, I see all these dependencies are correctly specified in the package. Except setuptools. Why do we need to specify them here too?

Reply:
Numpy temporarily vendored meson-python on this release.